### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by serialferris**
-added popup window

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -2,21 +2,181 @@ const hamburger = document.querySelector('.bars');
...
+    <button class="see-project-button" type="button">See Project</button>
+  </div>
+</div>`;
+
+workS.appendChild(
+  ...ConvertStringToHTML(featuredHtml).children,
+);
+
+const projectUnorderedList = document.createElement('ul');
+projectUnorderedList.className = 'random-projects';
```
Comment (leonardodiasb) [inline script.js:120]: - [ ] When the user hover over the random-projects list, all of the projects are getting larger and part of them are going out of the screen and overlapping other content, creating a bad user experience. Please fix that.
Your design:
image

File: style.css [hunk 2]
```diff
@@ -513,6 +522,133 @@ nav.active {
...
+  gap: 2.4rem;
+  width: max-content;
+  height: 90vh;
+  max-height: max-content;
+  margin: auto;
+  padding: 1.6rem;
+  background-color: white;
+  z-index: 5;
+  overflow-y: scroll;
+}
```
Comment (leonardodiasb) [inline style.css]: - [ ] Your modal window is not responsive in both mobile and desktop versions. The content inside does not fit inside the popup window in some screen sizes. Kindly fix that.
Your design:
Desktop -
image
Mobile -
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @serialferris, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README is descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (serialferris) [reply script.js:120]: I have removed the hover effect because of which the random projects were enlarged on hovering

File: style.css (hunk 2 above)
Comment (serialferris) [reply style.css]: I have made changes to fit the content in the popup

File: style.css [hunk 3]
```diff
@@ -509,8 +518,131 @@ nav.active {
...
```
Comment (leonardodiasb) [inline style.css]: - [ ] Your mobile popup cannot contain the entire content and still has a scrollbar resulting in a bad experience for the user. Take a deep look into the font sizes, paddings, margins and all the dimensions in the Figma design and try to replicate that on your project.
Untitled

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @serialferris, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You did a great job fixing some issues mentioned by the last review ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @serialferris , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions
- [ ] You could align the bottom buttos in the mobile view to look like the ones in the given design
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**